### PR TITLE
MAINT: Reverted the changes in #3300 that broke the MINIMIUM_REQUIREMENTS tests

### PR DESCRIPTION
**Pull request by hmaarrfk**
## Description
Bumping the minimum requirements seems to require more thought and care than just changing the Matplotlib version as shown by two attempts:

  * https://github.com/scikit-image/scikit-image/pull/3400
  * https://github.com/scikit-image/scikit-image/pull/3426

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [x] Unit tests

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by jni** [comment]
LOL GitHub now labels codecov comments as spam??

**Comment by hmaarrfk** [comment]
I like to label them.

File: doc/examples/features_detection/plot_local_binary_pattern.py [hunk 1]
```diff
@@ -166,9 +166,9 @@ def match(refs, img):
     best_name = None
     lbp = local_binary_pattern(img, n_points, radius, METHOD)
     n_bins = int(lbp.max() + 1)
-    hist, _ = np.histogram(lbp, density=True, bins=n_bins, range=(0, n_bins))
+    hist, _ = np.histogram(lbp, normed=True, bins=n_bins, range=(0, n_bins))
```
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply doc/examples/features_detection/plot_local_binary_pattern.py]: The use of `density` **requires** an upgraded MPL. but nobody wants to put in time to fix the build issues that arise from updating MPL and not other dependencies.

[the remark this answers is not in the document]
Comment (jni) [reply doc/examples/features_detection/plot_local_binary_pattern.py]: @soupault look at the referred PRs. @sciunto and @hmaarrfk have made repeated efforts to get Travis to pass with `density=True`, without success. Travis is passing with this change, so I favour merging, and not worrying about this until it breaks.

**Comment by emmanuelle** [comment]
Merging, since it seems to be the best we can do at the moment.

File: doc/examples/features_detection/plot_local_binary_pattern.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (soupault) [reply doc/examples/features_detection/plot_local_binary_pattern.py]: Thanks for the comments!

@jni I've seen those PRs, but from them it is really not obvious what is the root issue. Will take a look by myself (#3430).

**Pull request merged**